repository-data/application/src/main/resources/hcm-config/namespaces/nationalindicatorlibrary/indicator.yaml

```diff
@@ -54,6 +64,14 @@ definitions:
         jcr:primaryType: editor:templateset
       /hipposysedit:nodetype:
         /hipposysedit:nodetype:
+          /attachments:
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: true
+            hipposysedit:ordered: false
+            hipposysedit:path: publicationsystem:Attachments-v3
```

Comment:
This can be `nationalindicatorlibary:attachments` the v3 is only relevant to publictions